common/src/main/java/com/wynntils/mc/mixin/CapeLayerMixin.java

```diff
@@ -40,4 +53,32 @@ private void render(
             ci.cancel();
         }
     }
+
+    @ModifyArg(
+            method = "render",
```

Comment:
One other quirk we have in Wynntils is that we use "semi-qualified" mixin method naming. This is not common in the scene, but basically it gives us all the benefits of using fully-qualified method names, but the code stays readable. You can check the format  
from other mixins, but it's basically a fully qualified method target, without the "Lnet/.../MixinClass" prefix.

Don't forget to change both of the mixins in the PR. Thanks!

Reply:
Done!